CHANGELOG.md

```diff
@@ -4,6 +4,223 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.27.0] - 2023-08-18
+
+## 🚀 Features
+
+### add a metric tracking coprocessor latency ([Issue #2924](https://github.com/apollographql/router/issues/2924))
+
+Introduces a new metric for the router:
+
+```
+apollo.router.operations.coprocessor.duration
+```
+
+It has one attributes:
+
+```
+coprocessor.stage: string (RouterRequest, RouterResponse, SubgraphRequest, SubgraphResponse)
+```
+
+It is an histogram metric tracking the time spent calling into the coprocessor
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/3513
+
+### Supergraph coprocessor implementation ([PR #3408](https://github.com/apollographql/router/pull/3408))
+
+This adds support for coprocessors at the supergraph service level. Supergraph plugins work on the request side with a parsed GraphQL request object, so the query and operation name, variables and extensions are directly accessible. On the response side, they handle GraphQL response objects, with label, data, path, errors, extensions. The supergraph response contains a stream of GraphQL responses, which can contain multiple elements if the query uses `@defer` or subscriptions. When configured to observe the responses, the coprocessor will be called for each of the deferred responses.
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/3408
+
+### Configure AWS sigv4 authentication for subgraph requests ([PR #3365](https://github.com/apollographql/router/pull/3365))
```

Comment:
I don't know if there's a right answer here, but I'll go with what AWS _seems_ to do most often here in terms of capitalization.

```suggestion
### Configure AWS SigV4 authentication for subgraph requests ([PR #3365](https://github.com/apollographql/router/pull/3365))
```